docs/script-api.md

```diff
@@ -633,6 +633,16 @@ Returns unique id for the callback to be used in [clear_entity_callback](#clear_
 Sets a callback that is called right when a container is opened via up+door.
 The callback signature is `nil on_open(Entity self, Entity opener)`
 Use this only when no other approach works, this call can be expensive if overused.
+### [`set_pre_collision1`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=set_pre_collision1)
+`optional<CallbackId> set_pre_collision1(int uid, function fun)`<br/>
+Returns unique id for the callback to be used in [clear_entity_callback](#clear_entity_callback) or `nil` if uid is not valid.
+Sets a callback that is called right before the collision 1 event, return `true` to skip the game's collision handling.
+Use this only when no other approach works, this call can be expensive if overused.
+### [`set_pre_collision2`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=set_pre_collision2)
```

Comment:
what's the difference between set_pre_collision1 and set_pre_collision2 ?

Reply:
As I will explain in the update channel when the PR gets merged: it hasn't been investigated yet, so 1 and 2 are just temporary names until we have a clear image what the difference is. So the community can help with determining what they do.